test/e2e/e2e.spec.js

```diff
@@ -6,7 +6,7 @@ describe('/ (Verdaccio Page)', () => {
   // this might be increased based on the delays included in all test
   jest.setTimeout(20000);
 
-  const clickElement = async function(selector, options = { button: 'middle', delay: 100 }) {
```

Comment:
no idea the reason, but this middle is causing some issues

Reply:
Mmmmm ... ok, let me dive a bit more into it, the thing is works locally 😢 .... thanks for the hint @priscilawebdev 👍